Elevel/Application/Features/ReportCommands/CreateReportCommand.cs

```diff
@@ -80,8 +84,46 @@ public Handler(IApplicationDbContext context, IMapper mapper)
 
             public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
             {
+                Guid creatorId = Guid.Empty;
+
                 var report = _mapper.Map<Report>(request);
                 
+                if (request.AuditionId == null && request.QuestionId != null)
+                {
+                    var question = await _context.Questions.FirstOrDefaultAsync(x => x.Id == request.QuestionId, cancellationToken: cancellationToken);
```

Comment:
in that case ideally load just 1 field instead of loading full question. Just Select(x=> x.CreatorId). 
if don't want please add AsNoTracking

Reply:
Done